PluginManager/po/it-local.po

```diff
@@ -0,0 +1,265 @@
+# Italian translation for Gramps PluginManager addon
+#
+# This file is distributed under the same license as the Gramps package.
+#
+# Giansalvo Gusinu <giansalvo.gusinu@gmail.com>, 2019.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: \n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2019-01-22 09:28-0600\n"
+"PO-Revision-Date: 2019-02-10 03:00+0100\n"
+"Language: it\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"Last-Translator: Giansalvo Gusinu\n"
+"Language-Team: \n"
+"X-Generator: Poedit 2.2.1\n"
+
+#: PluginManager/PluginManager.gpr.py:29
+msgid "Plugin Manager Enhanced"
+msgstr "Gestore Plugin Avanzato"
+
+#: PluginManager/PluginManager.gpr.py:30
+msgid "An Addon/Plugin Manager with several additional capabilities"
+msgstr "Un Gestore di Addon/Plugin con varie funzionalità addizionali"
+
+#: PluginManager/PluginManager.py:87
+msgid "Plugin Manager - Enhanced"
+msgstr "Gestore dei Plugin - Avanzato"
+
+#: PluginManager/PluginManager.py:144
+msgid "_Help"
+msgstr "_Aiuto"
+
+#: PluginManager/PluginManager.py:151
+msgid ""
+"Enter search words to filter the addons.\n"
+"All the words must be present somewhere in the row or\n"
+"the addon filename to be included in the search.\n"
+"Word case and order is ignored."
+msgstr ""
+"Inserisce le parole da cercare per filtrare gli addons.\n"
+"Tutte le parole devono essere presenti da qualche parte nella riga oppure\n"
+"il nome dell'addon da includere nella ricerca.\n"
+"Maiuscole/minuscole e l'ordine delle parole sono ignorate."
+
+#: PluginManager/PluginManager.py:155
+msgid "Search..."
+msgstr "Ricerca..."
+
+#: PluginManager/PluginManager.py:161
+msgid "Check for updated addons now"
+msgstr "Cerca gli addons aggiornati adesso"
+
+#: PluginManager/PluginManager.py:168
+msgid "Reload"
+msgstr "Ricarica"
+
+#: PluginManager/PluginManager.py:174 PluginManager/PluginManager.py:735
+msgid "_Close"
+msgstr "_Chiudi"
+
+#: PluginManager/PluginManager.py:207
+msgid "Restart..."
+msgstr "Riavvia..."
+
+#: PluginManager/PluginManager.py:208
+msgid ""
+"Please Restart Gramps so that your addon changes can be safely completed."
+msgstr ""
+"Si prega di Riavviare Gramps in modo da completare l'aggiornamento degli "
```

Comment:
Without "si prega", the "please" form is English only:
Riavviare Gramps in modo da completare l'aggionramento dei componenti aggiuntivi in maniera sicura.